sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2ScanExecBase.scala

```diff
@@ -46,6 +47,31 @@ trait DataSourceV2ScanExecBase extends LeafExecNode {
     Utils.redact(sqlContext.sessionState.conf.stringRedactionPattern, result)
   }
 
+  /**
+   * Shorthand for calling redactString() without specifying redacting rules
```

Comment:
nit: I don't see a function named `redactString()` around here.

Reply:
@Ngone51 Thanks.. have changed it.